src/actions/errors.js

```diff
@@ -0,0 +1,14 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
I know that a new file for this looks weird.
I initially moved this to `actions/app` because it made sense that it could be more global, but this triggered some circular dependency (`actions/app` imports things from `actions/receive-profile`). I could leave it in `receive-profile` but I thought this was weird...

Reply:
I see. I didn't super like the new file either but previous file was worse. So this is definitely an improvement 😄